Firestore/Example/Tests/SpecTests/FSTSpecTests.mm

```diff
@@ -770,17 +773,24 @@ - (void)testSpecTests {
   // Enumerate the .json files containing the spec tests.
   NSMutableArray<NSString *> *specFiles = [NSMutableArray array];
   NSMutableArray<NSDictionary *> *parsedSpecs = [NSMutableArray array];
-  NSBundle *bundle = [NSBundle bundleForClass:[self class]];
-  NSFileManager *fs = [NSFileManager defaultManager];
   BOOL exclusiveMode = NO;
-  for (NSString *file in [fs enumeratorAtPath:[bundle resourcePath]]) {
-    if (![@"json" isEqual:[file pathExtension]]) {
+
+  auto source_file = Path::FromUtf8(__FILE__);
```

Comment:
Hmm, I understand it would work on a Mac, but does it work on iOS simulator? What about a real device?

Reply:
Somewhat surprisingly (as the travis results show), this works just fine in a simulator. It wouldn't work on a real device though.

Unfortunately, I wasn't able to get resource bundling to work via CMake so there's currently no path to getting this to work on a real device using that build. I've added a TODO for the moment.